android/build.gradle

```diff
@@ -31,10 +31,10 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('Hmssdk_compileSdkVersion', 30)
+    compileSdkVersion safeExtGet('Hmssdk_compileSdkVersion', 31)
     defaultConfig {
         minSdkVersion safeExtGet('Hmssdk_minSdkVersion', 21)
-        targetSdkVersion safeExtGet('Hmssdk_targetSdkVersion', 29)
+        targetSdkVersion safeExtGet('Hmssdk_targetSdkVersion', 31)
```

Comment:
@stanwolverine Similar to above, why is updating Target SDK version required?
Clients who have not updated to 31 won't be able to compile the app.